components/Common/CrossLink/index.tsx

```diff
@@ -0,0 +1,39 @@
+import styles from './index.module.scss';
+import classNames from 'classnames';
+import LocalizedLink from '@/components/LocalizedLink';
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/solid';
+import { FormattedMessage } from 'react-intl';
+import type { FC } from 'react';
+
+type CrossLinkProps = {
+  type: 'previous' | 'next';
+  text: string;
+  url: string;
+};
+
+const CrossLink: FC<CrossLinkProps> = ({ type, text, url }) => (
+  <LocalizedLink className={styles.crossLink} href={url}>
+    <div
+      className={classNames(styles['header'], {
+        [styles['reverse']]: type === 'next',
+      })}
+    >
+      {type === 'previous' ? (
```

Comment:
I checked the Figma again but I don't think we have another usage of chevron arrows. The Pagination component uses normal arrows (the ones with a tail).

Thus I put it inside CrossLink's `index.tsx` for now.

We can have it covering 4 directions (+ up & down) as well but I think let's not overthink it for now.

Reply:
Afaik the Button Component uses chevron arrows. At least if you click on Button on Figma you can see that it has a placeholder for that.

It might be good to put it in a separate component explicitly named PrevNextArrows imo

Reply:
Saw the placeholder.

Added `PrevNextArrow` (singular). Thinking about it for some time, I agree that it's cleaner this way.